### DISCUSSION TITLE
Kunena versie 6.1.3.1 plugin for uddeim

### DISCUSSION THREAD

**Discussion by Leen29** [Kunena/Kunena-Forum#9557] [Q&A]
i am testing this kunena plugin for uddeim download from
 https://github.com/Kunena/Kunena-Forum/tree/K6.1/src/plugins/plg_kunena_uddeim
the uddeim.php  have on line 35 at the end says ", KPATH_ADMIN"if i remove that the plugin works in kunena oke
only when I activate it do I get an error at administrator > Components > uddeIM PMS as on the picture "plugin_on.jpg"
. but pmss does work on the front. picture "pmwork.jpg"
If I turn off the plugin again, it will show up properly in Administrator side > Components > UddeIM Administration
picture  "pluginoff.jpg"
how can i fix or get that better?
I hope this is the right place to ask this question, if not let me know where I can ask
thank you
![plugin_on](https://github.com/Kunena/Kunena-Forum/assets/17086048/15b84758-349a-46a6-a78a-71d1b4a4ead0)
![pmworks](https://github.com/Kunena/Kunena-Forum/assets/17086048/154a52db-6647-4168-b7b3-fe93e7ccc2dc)
![pluginoff](https://github.com/Kunena/Kunena-Forum/assets/17086048/3f926a8c-d44e-4213-a015-fce25fb5e835)

**Comment by xillibit** [comment]
Hello,

The uddeim plugin integration is present since the K6.1.3 release, i have seen already couple of issues when KPATH_ADMIN was used. I will look what can be done

**Comment by Leen29** [comment]
Thanks 
yes  maybe on github already present since the K6.1.3
but I hadn't looked there in the normal download version
6.1.3.1 I don't see it there.
so only testing with the one from github since today

**Comment by Leen29** [comment]
if I use the original plg_kunena_uddeim.zip from github, I get the message as attached image at the current test forum with gantry.
if I remove that ", KPATH_ADMIN" again, the site will be okay again.
What surprises me more is that I converted to another
forum was able to install and use the original plg_kunena_uddeim.zip without removing datkpath_admin.
So it seems to me that it may also be due to the server settings?
and now tested where it goes well right away PHP Version 7.4.33 is set
and when I test PHP Version 7.4.33 on my local pc it works fine
as soon as i set PHP Version 8.0.26 or higher PHP Version 8.1.17 then KPATH_ADMIN ERROR comes up
![veroorzaakthauddimplugindit](https://github.com/Kunena/Kunena-Forum/assets/17086048/fc50f231-df04-4f74-a3b4-e8777cd13d94)

**Comment by xillibit** [comment]
> Thanks
> yes maybe on github already present since the K6.1.3
> but I hadn't looked there in the normal download version
> 6.1.3.1 I don't see it there.
> so only testing with the one from github since today

I had seen yerstayday that i had forgot something so the uddeim wasn't inclued in the build, for the next releases it will be here
